### ISSUE TITLE
Allow configuration for excluded paths

### ISSUE THREAD

**Issue by felixLam** [stil4m/elm-analyse#34]
It would be nice to be able to exclude certain paths. E.g. I would like to exclude the "elm-stuff" directory as well as our "vendor" directory as this includes code that is not 100% under our control.

Ideally this could be done via the `elm-analyse.json`. E.g.:

```json
{
    "checks": {
        "LineLengthExceeded" : false
    },
    "excludedPaths": [ "elm-stuff", "vendor" ]
}

```

**Comment by stil4m** [comment]
Hi Felix,
Thanks for your report. This would be a great addition. 

The `elm-stuff` should be ignored by default. It only analyses the dependencies to determine the interfaces and does this only once. Do you encounter problems with this?

**Comment by felixLam** [comment]
I might have been confused by the load logs.

**Comment by stil4m** [comment]
Check, I understand why. I've created a ticket that should fix that as well #35.

**Comment by felixLam** [comment]
Thanks for your work on this. Really happy to have found this and elm-analyse is now part of our CI setup and runs on every PR.
